docs/sharded-computation.ipynb

```diff
@@ -268,40 +292,7 @@
     "id": "UEObolTqw4pp"
    },
    "source": [
-    "The device numbers here are not in numerical order, because the mesh reflects the underlying toroidal topology of the device.\n",
-    "\n",
-    "The {class}`~jax.sharding.NamedSharding` includes a parameter called `memory_kind`. This parameter determines the type of memory to be used and defaults to `device`. You can set this parameter to `pinned_host` if you prefer to place it on the host.\n",
-    "\n",
-    "To create a new sharding that only differs from an existing sharding in terms of its memory kind, you can use the `with_memory_kind` method on the existing sharding."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/"
-    },
-    "id": "aKNeOHTJnqmS",
-    "outputId": "847c53ec-8b2e-4be0-f993-7fde7d77c0f2"
-   },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "pinned_host\n",
-      "device\n"
-     ]
-    }
-   ],
-   "source": [
-    "s_host = jax.NamedSharding(mesh, P('x', 'y'), memory_kind='pinned_host')\n",
-    "s_dev = s_host.with_memory_kind('device')\n",
-    "arr_host = jax.device_put(arr, s_host)\n",
-    "arr_dev = jax.device_put(arr, s_dev)\n",
-    "print(arr_host.sharding.memory_kind)\n",
-    "print(arr_dev.sharding.memory_kind)"
+    "The device numbers here are not in numerical order, because the mesh reflects the underlying toroidal topology of the device."
```

Comment:
Is this not running on TPU anymore? I'm confused why it is saying toroid.

Reply:
Ohh, good point. Let me remove that.